systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -87,6 +87,7 @@ public abstract class AbstractST implements TestSeparator {
     protected static Map<String, String> mapWithTestTopics = new HashMap<>();
     protected static Map<String, String> mapWithTestUsers = new HashMap<>();
     protected static Map<String, String> mapWithScraperNames = new HashMap<>();
+    protected static Map<String, Long> mapWithTestExecutionStartTimes = new HashMap<>();
```

Comment:
I wonder if we really need `map` in the name when it is kinda obvious from type :D

Reply:
i can rename this, but just wanted to hold current naming for now. Had a discussion with morsak and we agreed these maps could be moved to storage in another PR which could make it simpler.